nbs/docs/Prediction_intervals_in_forecasting_models.ipynb

```diff
@@ -0,0 +1,3012 @@
+{
```

Comment:
<p>This plot is still being shown. You may have to call plt.close() to avoid it showing</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/mlforecast/pull/196/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>